nibabel/freesurfer/mghformat.py

```diff
@@ -255,7 +255,7 @@ def get_zooms(self):
         .. _mghformat: https://surfer.nmr.mgh.harvard.edu/fswiki/FsTutorial/MghFormat#line-82
         '''
         # Do not return time zoom (TR) if 3D image
-        tzoom = (self['tr'],)[:self._ndims() > 3]
+        tzoom = (self['tr'],)[:int(self._ndims() > 3)]
```

Comment:
I think I wrote this, but looking at it, I think it's only a couple characters more to be a lot clearer in intent:

```Python
tzoom = (self['tr'],) if self._ndims() > 3 else ()
```